zio-http/src/test/scala/zhttp/middleware/MiddlewareSpec.scala

```diff
@@ -166,6 +167,41 @@ object MiddlewareSpec extends DefaultRunnableSpec with HttpAppTestExtensions {
               res <- app(request)
             } yield assert(res.getHeadersAsList)(hasSubset(expected))
           }
+      } +
+      suite("addCookie middleware") {
+        testM("should add set-cookie header") {
+          val app: Http[Any, Nothing, Request, Option[String]] =
+            (Http.ok @@ addCookie(Cookie("test", "testValue"))).getHeader(HttpHeaderNames.SET_COOKIE.toString)
```

Comment:
`HttpHeaderNames.SET_COOKIE.toString`  can we directly use string instead? Also I assume this will be case Insensitive so using string directly might not have any issue